package.json

```diff
@@ -19,6 +19,7 @@
     "@defillama/sdk": "latest",
     "@project-serum/anchor": "^0.25.0",
     "@solana/spl-token": "^0.3.0",
+    "@solana/spl-stake-pool": "0.6.3",
```

Comment:
Your Note from defilama says not to edit or push package.json, not sure if that’s relevant to this change